### DISCUSSION TITLE
Atmos support

### DISCUSSION THREAD

**Discussion by mtweeman** [jdx/mise#2540] [Ideas]
Hey,

there are plenty of Terraform wrappers supported already by mise, like Terragrunt, Terramate, etc.
Would it be possible to add support for [Atmos](https://github.com/cloudposse/atmos)?
I'm not familiar with this repo = not sure how much of an effort that is. If this is as simple as adding corresponding file in registry/plugins/ I can raise the PR.

Thanks for help!

**Comment by Marukome0743** [comment]
Atmos is now available directly through mise.

Support was added in #2577 and released in mise v2024.9.4. The current registry maps the `atmos` shorthand to the Aqua backend, so it can be installed with:

```sh
mise use atmos
# or explicitly:
mise use aqua:cloudposse/atmos
```

I also verified this against the current main branch in a clean mise data directory on macOS arm64. mise installed Atmos 1.224.1, and `atmos version` completed successfully:

```text
👽 Atmos 1.224.1 on darwin/arm64
```

The idiomatic version file `.atmos-version` is supported as well.

*AI-assisted — Tool: Codex; model: OpenAI/GPT-5; version: unavailable.*
